test/src/sdk-harness/test_projects/evm/mod.rs

```diff
@@ -0,0 +1,35 @@
+use fuels::{prelude::*, tx::ContractId};
+
+abigen!(
+    EvmTestContract,
+    "test_projects/evm/out/debug/evm-abi.json"
+);
+
+async fn get_evm_test_instance() -> (EvmTestContract, ContractId) {
+    let wallet = launch_provider_and_get_single_wallet().await;
+    let id = Contract::deploy(
+        "test_projects/evm/out/debug/evm.bin",
+        &wallet,
+        TxParameters::default(),
+    )
+    .await
+    .unwrap();
+    let instance = EvmTestContract::new(id.to_string(), wallet);
+
+    (instance, id)
+}
+
+#[tokio::test]
+async fn can_call_from() {
```

Comment:
This should be split into 2 tests. How do you know which one will fail if this test fails? They are two separate functions. This makes it harder to debug

Reply:
Done.